docs/style-guide.md

```diff
@@ -27,7 +28,9 @@ This document contains background on how the style decisions were reached.  The
             - [Pros](#pros-3)
             - [Cons](#cons-3)
             - [Decision](#decision-3)
-
+    - [Element Ordering](#element-ordering)
+	      - [General Guidelines](#general-guidelines)
+	      - [resolvedOptions](#resolvedoptions)
```

Comment:
The extra indentation here breaks Markdown list output: https://github.com/tc39/ecma402/blob/97ba211cddb8a4a22f67e327af05372b545f2bcd/docs/style-guide.md#table-of-contents:~:text=Element%20Ordering
```suggestion
    - [Element Ordering](#element-ordering)
	    - [General Guidelines](#general-guidelines)
	    - [resolvedOptions](#resolvedoptions)
```